src/uniprotkb/components/entry/tabs/genomic-coordinates/GenomicLoc.tsx

```diff
@@ -0,0 +1,48 @@
+import { LongNumber } from 'franklin-sites';
+
+import ExternalLink from '../../../../../shared/components/ExternalLink';
+
+import { GenomicLocation } from './types';
+
+export const getEnsemblLink = (
+  taxID: number,
+  chromosome?: string,
+  start?: number,
+  end?: number
+) =>
+  `https://www.ensembl.org/${taxID}/Location/View?r=${chromosome}:${start}-${end}`;
+
+type GenomicLocProps = {
+  genomicLocation: GenomicLocation;
+  taxID: number;
+};
+
+const GenomicLoc = ({ genomicLocation, taxID }: GenomicLocProps) => (
+  <ExternalLink
+    url={getEnsemblLink(
+      taxID,
+      genomicLocation.chromosome,
+      genomicLocation.reverseStrand
+        ? genomicLocation.end
+        : genomicLocation.start,
+      genomicLocation.reverseStrand
+        ? genomicLocation.start
+        : genomicLocation.end
+    )}
+  >
+    {genomicLocation.chromosome}:
+    <LongNumber>
+      {(genomicLocation.reverseStrand
+        ? genomicLocation.end
+        : genomicLocation.start) ?? ''}
+    </LongNumber>
+    -
```

Comment:
Same as previous suggestion about improving readability
```suggestion
    {' - '}
```

Reply:
Covered in https://github.com/ebi-uniprot/uniprot-website/commit/52b9fdeabfa507029e73e580e51982ecae7f595a